src/AppBundle/Form/ServerType.php

```diff
@@ -17,6 +17,7 @@ public function buildForm(FormBuilderInterface $builder, array $options)
         $builder
             ->add('ip')
             ->add('os')
+            ->add('hostedOn')
```

Comment:
Lets make this a dropdown, with options 'AWS', 'Rackspace', 'Digital Ocean', 'Other'